### ISSUE TITLE
Diagnosis Standards Page does not match the latest Schema

### ISSUE THREAD

**Issue by ecerami** [ncihtan/htan-portal#378]
For the DFCI HTAN Atlas, I created a template for Clinical Diagnosis via the Data Curator:

https://docs.google.com/spreadsheets/d/1jeN7ELRLuKS_V1yUT4buWi-BEMW3ZmkdVfDbFMr8fUo/edit#gid=0

If you scroll all the way to the right, you will see two columns:

- Days to Progression Free
- Days to Recurrence

Now, if I go to:  https://data.humantumoratlas.org/standard/clinical

and open the Diagnosis panel, there is no mention of either of these fields.  Perhaps, the portal is using an older version of the schema?

**Comment by ecerami** [comment]
I wanted to see if we can increase the priority on this one?

It seems to affect no only the diagnosis page, but the biospecimen page as well.

For biospecimens, for example, the Data Curator Spreadsheet includes:  Source HTAN Biospecimen ID, but this is not mentioned in the biospecimen standards table:  https://humantumoratlas.org/standard/biospecimen

**Comment by inodb** [comment]
Yes this has been an issue for a while: https://github.com/ncihtan/htan-portal/issues/295. Will try and get it sorted

**Comment by ecerami** [comment]
thanks, Ino.

**Comment by vthorsson** [comment]
@ecerami , @inodb This may be related to something we have been trying to figure  out, namely how to reproduce the columns of the google sheet generated by the Data Curator App, from the schema csv. It looks like we may have (at last!) been able do this, and have some notebooks illustrating the procedure to share.

For Diagnosis in  particular
-Days to Progression Free
-Days to Recurrence
are obtained in 2nd step of the procedure.  One thing that might be happening is that the data portal is listing only what is generated in the 1st step ( I saw evidence of that for Biospecimen as well). Thus versioning may be a factor, but I suspect it is not the main sources of differences.

( + @PozhidayevaDarya )

**Comment by ecerami** [comment]
@inodb or @milen-sage sorry to nudge, but any updates on this issue?  need it to complete the missing manual doc.

**Comment by milen-sage** [comment]
@ecerami I shared a snippet that should help re-populating the pages more accurately with @inodb and @PozhidayevaDarya. @PozhidayevaDarya has tested the snippet and it produces the expected output. Based on my understanding, @inodb is working on integrating this snippet in the portal metadata processing step and then updating the generation of the standards pages.

**Comment by ecerami** [comment]
just a little update here:  i modified the HTAN Portal code to now point to the current JSON LD under:  https://github.com/ncihtan/data-models.  This is now running at:  https://htan-portal-nextjs-git-standards-docs-htan.vercel.app/standards.  Was hoping that this would fix the issue, but alas, it's probably more complicated...
